include/swift/AST/DiagnosticsSema.def

```diff
@@ -4741,7 +4741,9 @@ ERROR(global_actor_non_unsafe_init,none,
       "global actor attribute %0 argument can only be '(unsafe)'", (Type))
 ERROR(global_actor_non_final_class,none,
       "non-final class %0 cannot be a global actor", (DeclName))
-      
+ERROR(global_actor_top_level_var,none,
+      "top-level code variables cannot have a global actor", ())
```

Comment:
add a test to cover this? there doesn't seem to be one

Reply:
Done 

https://github.com/apple/swift/pull/40998/files#diff-0b3cf9b84143abf54688a2441cb7d2425cb6819e229078be58f610ba2f09c1c7R6-R8